terraform/main.tf

```diff
@@ -566,6 +566,14 @@ module "agent-monitor" {
   automation_destination_id = module.destination["message-queue"].destination_id
 }
 
+module "app-monitor" {
+  source = "./app-monitor-module"
+  for_each = { for a in jsondecode(file("./app-alert.json")): a.name => a }
+  app_monitor = each.value
+  depends_on = [aws_opensearch_domain.es]
+  automation_destination_id = module.destination["wso2-alert"].destination_id
```

Comment:
What if you don't want to use this destination?

JSON probably needs to include a bunch more details like the destination (interval, throttle, action(s), templates)

Reply:
Use yaml as recommended and use queue name for destination.